kuma/wiki/jinja2/wiki/revision.html

```diff
@@ -56,7 +56,7 @@ <h1>{{ _('Revision %(id)s of <a href="%(url)s">%(title)s</a>'|safe, id=revision.
     <details open="open">
       <summary><h2>{{ _('Revision Content') }}</h2></summary>
       <article id="wikiArticle" class="text-content">
-        {{ revision.content_cleaned|safe }}
+        {{ revision.content|safe }}
```

Comment:
There are XSS attempts in historical revisions that need to be cleaned, such as the ones reported in [bug 821988](https://bugzilla.mozilla.org/show_bug.cgi?id=821988). We may be able to assume clean content going forward from when this ships, but we can't trust current content.

To emulate an old revision in the dev environment:

1. Edit a local page, get the revision ID
2. Run this in the Django shell:

```
rev_id = 123456 # Or whatever your ID is
content = '<p onclick="alert(1)">Click me!</p>'
Revision.objects.filter(id=rev_id).update(content=content)
```
3. Load the revision, click "Click me!"

Reply:
Oh man, good catch, sorry I missed this!